src/pip/_internal/commands/inspect.py

```diff
@@ -94,4 +94,11 @@ def _dist_to_dict(self, dist: BaseDistribution) -> Dict[str, Any]:
         # requested
         if dist.installed_with_dist_info:
             res["requested"] = dist.requested
+        # convert project_url to dict
+        if "project_url" in res["metadata"]:
+            project_url_dict = {}
+            for i in res["metadata"]["project_url"]:
+                name, url = i.split(", ", 1)
```

Comment:
What if the value is incorrectly formatted? This will raise a ValueError. We shouldn't fail on invalid metadata, as there's a lot of legacy packages around that don't necessarily conform to the latest standards.

Reply:
Fair Point. I haven't though about this. Maybe we should ignore the URL in that case?

Reply:
Ignoring sounds reasonable, or alternatively we could fallback to just use the entire value as the key.